modules/036-kube-proxy/images/kube-proxy/werf.inc.yaml

```
@@ -2,9 +2,14 @@
 {{- $iptables_image_version := $iptables_version | replace "." "-" }}
 {{- $binaries := "/usr/sbin/conntrack /usr/lib64/libnetfilter_conntrack.so*" }}
 ---
-artifact: {{ .ModuleName }}/{{ $.ImageName }}-artifact
-from: {{ $.Images.BASE_ALT_DEV }}
+image: {{ $.ModuleName }}/{{ $.ImageName }}-artifact
+fromImage: common/relocate-artifact
+final: false
 shell:
+  beforeInstall:
+    - apt-get update
+    - apt-get install conntrack-tools -y
+    - rm -rf /var/lib/apt/lists/* /var/cache/apt/* && mkdir -p /var/lib/apt/lists/partial /var/cache/apt/archives/partial
```

need apply after merge refactored relocate-artifact in main
```suggestion
    - apt-get install conntrack-tools -y
```